src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig

```diff
@@ -55,6 +55,12 @@
                 <span>{{ collector.route|default('n/a') }}</span>
             </div>
 
+            {%- set keystemplates = profile.collectors['twig'].getTemplates|keys|default(false) -%}
```

Comment:
this creates a dependency with the twig collector which can [break](https://github.com/symfony/symfony/blob/7b03bff57a2782edc6558d57df7ccff99ed0f1a9/src/Symfony/Component/HttpKernel/Profiler/Profile.php#L242), to me that implies it belongs in the twig toolbar block at least.